### PR TITLE
Add getting started docs for permissions (for adopters)

### PR DESCRIPTION
**Pull request by Joonpark13**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: docs/permission/getting-started.md [hunk 1]
```diff
@@ -3,3 +3,133 @@ id: getting-started
 title: Getting Started
 description: How to get started with the permission framework as an integrator
 ---
+
+If you prefer to watch a video instead, you can start with this video introduction:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/EQr9tFClgG0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (freben) [inline docs/permission/getting-started.md:9]: 🌟 🌟 🌟 🌟

File: docs/permission/getting-started.md [hunk 2]
```diff
@@ -3,3 +3,133 @@ id: getting-started
...
+
+Backstage integrators control permissions by writing a policy. In general terms, a policy is simply an async function which receives a request to authorize a given permission for a user and (optional) resource, and returns a decision on whether to authorize that permission. Integrators can implement their own policies from scratch, or adopt reusable policies written by others.
```
Comment (freben) [inline docs/permission/getting-started.md:11]: the wording "authorize a permission" chafes a bit - "authorize an action"? But maybe that goes against the vernacular we've set overall

File: docs/permission/getting-started.md [hunk 3]
```diff
@@ -3,3 +3,133 @@ id: getting-started
...
+
+## Prerequisites
+
+The permissions framework depends on a few other Backstage systems, which must be set up before we can dive into writing a policy.
+
+### Upgrade to the latest version of Backstage
+
+The permissions framework itself is new to Backstage and still evolving quickly. To ensure your version of Backstage has all the latest permission-related functionality, it’s important to upgrade to the latest version. The Backstage upgrade helper is a great tool to help ensure that you’ve made all the necessary changes during the upgrade!
```
Comment (freben) [inline docs/permission/getting-started.md:19]:
```suggestion
The permissions framework itself is new to Backstage and still evolving quickly. To ensure your version of Backstage has all the latest permission-related functionality, it’s important to upgrade to the latest version. The [Backstage upgrade helper](https://backstage.github.io/upgrade-helper/) is a great tool to help ensure that you’ve made all the necessary changes during the upgrade!
```

**Review by freben — APPROVED** [review]
NICE (wording tweaks but approving)

File: docs/permission/getting-started.md (hunk 2 above)
Comment (Joonpark13) [reply docs/permission/getting-started.md:11]: Changed to "authorize an action" as we've used that phrasing in a few other places.

File: docs/permission/getting-started.md [hunk 4]
```diff
@@ -3,3 +3,133 @@ id: getting-started
...
+```typescript
+import { IdentityClient } from '@backstage/plugin-auth-node';
+import { createRouter } from '@backstage/plugin-permission-backend';
+import { AuthorizeResult } from '@backstage/plugin-permission-common';
+import {
+  PermissionPolicy,
+  PolicyDecision,
+} from '@backstage/plugin-permission-node';
+import { Router } from 'express';
+import { PluginEnvironment } from '../types';
```
[the remark this answers is not in the document]
Comment (Joonpark13) [reply docs/permission/getting-started.md:53]: Oops, not sure where those went - must be a copy/paste error.

**Pull request merged**